src/Sylius/Bundle/ShopBundle/Resources/config/app/twig_hooks/account/address_book/create.yaml

```diff
@@ -1,20 +1,30 @@
 twig_hooks:
     hooks:
-        'sylius_shop.account.address_book.create':
+        'sylius_shop.account.address_book.create.layout':
+            breadcrumbs:
```

Comment:
Shouldn't breadcrumb be a part of header 🤔

Reply:
Header is part of the content and in that case breadcrumbs would be in a wrong position

Reply:
maybe we should then just add the header hook to be consistent with other places 🤔